app/views/collection_items/index.html.erb

```diff
@@ -13,12 +13,21 @@
 %>
 
 <ul class="navigation actions" role="navigation">
-  <li><%= span_if_current ts("Awaiting Approval"), base_path, !(params[:rejected] || params[:approved]) %></li>
+  <li><%= span_if_current ts("Awaiting Approval"), base_path, !(params[:rejected] || params[:approved] || params[:invited]) %></li>
+  <% unless @user %>
+    <li><%= span_if_current ts("Invited"),  base_path + "?invited=true"  %></li>
+  <% end %>
   <li><%= span_if_current ts("Rejected"), rejected_path %></li>
   <li><%= span_if_current ts("Approved"), approved_path %></li>
 </ul>
 <!--/subnav-->
 
+<% if params[:invited] %>
+  <p class="notice">
+    <%= ts("Works listed here have already been approved for this collection.") %>
```

Comment:
There's no logic or HTML differentiating these strings, so I think it can be combined as `<%= ts("Works listed here have already been approved for this collection. Once a work's owner has approved inclusion in this collection, the work will be moved to 'Approved'.") %>`